conf/config.yaml

```diff
@@ -82,6 +82,9 @@ data:
     tables:
       - name: employees.student
         allow_full_scan: true
+        auto_increment:
```

Comment:
How about using `sequence` instead of `auto_increment`?

Reply:
i think， auto